### PR TITLE
Improve Register page error message

### LINKED ISSUE #232: Registration Process Fails with Existing Credentials or Similar Email Addresses Without Specific Error Message
Specifically, the process fails under the following conditions:

1. **Existing User Credentials:** When trying to register with credentials that are already associated with an existing user account.

2. **Similar Email Addresses:** When the email entered in the second email field is similar to the one provided in the first email field.

In both scenarios, the application does not provide a specific error message indicating the cause of the failure. Instead, it only displays a generic message stating "something went wrong.

### PR DESCRIPTION
**Pull request by beesaferoot**
### Brief summary of the change made

The change adds a custom error message when the email address used already exists on the system.

Fixes #232

**Preview:** 

Screenshot 2024-11-23 at 10 45 54 PM

### Pull Request checklist

Please confirm you have completed any of the necessary steps below.

- [x] Meaningful Pull Request title and description
- [x] Changes tested as described above
- [ ] Added appropriate documentation for the change.
- [ ] Created GitHub issues for any relevant followup/future enhancements if appropriate.

### CODE REVIEW

File: src/frontend/src/assets/locales/en.json [hunk 1]
```diff
@@ -553,5 +553,9 @@
     "week": "Week | Weekly",
     "yes": "Yes",
     "yesterday": "Yesterday"
-  }
+  },
+  "errors": {
+     "alreadyUsedEmail": "Account with email already exists",
+     "emailUserDifferent": "Company and personal email should be different"
```
Comment (dmohns) [inline src/frontend/src/assets/locales/en.json:559]:
```suggestion
     "emailUserDifferent": "Company contact and personal email should be different"
```

File: src/frontend/src/assets/locales/bu.json [hunk 2]
```diff
@@ -553,5 +553,9 @@
     "week": "ရက်သတf ပတ် | အပတ်စV်",
     "yes": "ဟတု ်/မှန",
     "yesterday": "မေနက့ "
-  }
+  },
+  "errors": {
+     "alreadyUsedEmail": "Account with email already exists",
+     "emailUserDifferent": "Company and personal email should be different"
```
Comment (dmohns) [inline src/frontend/src/assets/locales/bu.json:559]:
```suggestion
     "emailUserDifferent": "Company contact and personal email should be different"
```

File: src/frontend/src/assets/locales/fr.json [hunk 3]
```diff
@@ -553,5 +553,9 @@
     "week": "Semaine | Hebdomadaire",
     "yes": "Oui",
     "yesterday": "Hier"
+  },
+  "errors": {
+    "alreadyUsedEmail": "Account with email already exists",
```
Comment (dmohns) [inline src/frontend/src/assets/locales/fr.json:558]:
```suggestion
    "alreadyUsedEmail": "Le compte avec cet e-mail existe déjà",
```

File: src/frontend/src/assets/locales/fr.json [hunk 4]
```diff
@@ -553,5 +553,9 @@
...
+    "emailUserDifferent": "Company and personal email should be different"
```
Comment (dmohns) [inline src/frontend/src/assets/locales/fr.json:559]:
```suggestion
    "emailUserDifferent": "L'email de contact de l'entreprise et l'email personnel doivent être différents"
```

**Review by dmohns — COMMENTED** [review]
Nice! Quick suggestion regarding the error message wordings

**Comment by devjhane** [comment]
Updated, kindly reviewed @dmohns

**Comment by beesaferoot** [comment]
Updated, kindly reviewed @dmohns

**Comment by beesaferoot** [comment]
@dmohns  Ready for review.

**Review by dmohns — APPROVED** [review]
Looks good, thank you :shipit:

**Pull request merged**